tests/FSharp.Compiler.Service.Tests/TooltipTests.fs

```diff
@@ -29,7 +31,8 @@ let testXmlDocFallbackToSigFileWhileInImplFile sigSource implSource line colAtEn
             SourceFiles = [| "A.fsi"; "A.fs" |] }
 
     let checker =
-        FSharpChecker.Create(documentSource = DocumentSource.Custom documentSource)
+        FSharpChecker.Create(documentSource = DocumentSource.Custom documentSource,
```

Comment:
Just to confirm - different CI legs will put different values to the switch hence will be testing both old and new compiler?

Reply:
Exactly.

I don't know about a better way of doing this other than rewriting each test to a theory with 2 options. (Which I did for some tests, but it's not really a scalable approach.)

Reply:
Yeah that's a valid approach, no objections here.